### PR TITLE
stuff, u know?!!

### PR DESCRIPTION
**Pull request by condret**

### CODE REVIEW

File: libr/core/cmd_section.c [hunk 1]
```diff
@@ -461,7 +462,10 @@ static int cmd_section(void *data, const char *input) {
 				sprintf (vname, "area%d", (int)ls_length (core->io->sections));
 				name = vname;
 			}
-			r_io_section_add (core->io, offset, vaddr, size, vsize, rwx, name, 0, fd);
+			s = r_io_section_add (core->io, offset, vaddr, size, vsize, rwx, name, 0, fd);
+			if (s) {
+				r_io_section_apply (core->io, s->id, R_IO_SECTION_APPLY_FOR_ANALYSIS);
```
Comment (radare) [inline libr/core/cmd_section.c]: This define looks strange as hell

Comment (condret) [inline libr/core/cmd_section.c]: what define?

Comment (XVilka) [inline libr/core/cmd_section.c]: R_IO_SECTION_APPLY_FOR_ANALYSIS - probably this one

Comment (condret) [inline libr/core/cmd_section.c]: R_TRUFAE would be better, i know

**Comment by radare** [comment]
Looks like alvaro already commit this

**Pull request closed without merging**